### PR TITLE
feat: add `victoria_metrics` module

### PR DESCRIPTION
**Pull request by level-one-learner**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -54,6 +54,9 @@ pub mod redis;
 #[cfg(feature = "trufflesuite_ganachecli")]
 #[cfg_attr(docsrs, doc(cfg(feature = "trufflesuite_ganachecli")))]
 pub mod trufflesuite_ganachecli;
+#[cfg(feature = "victoriametrics")]
+#[cfg_attr(docsrs, doc(cfg(feature = "victoriametrics")))]
```
Comment (DDtKey) [inline src/lib.rs:58]: What do you think of `victoria-metrics` as feature name?

**Review by DDtKey — COMMENTED** [review]
Thanks for the contribution! 🚀 

One question and LGTM

File: src/lib.rs (hunk 1 above)
Comment (level-one-learner) [reply src/lib.rs:58]: Sure, thank you. Do you know if I only need to change the two lines?

```rust
#[cfg(feature = "victoria-metrics")]
#[cfg_attr(docsrs, doc(cfg(feature = "victoria-metrics")))]
pub mod victoriametrics;
```

Comment (level-one-learner) [reply src/lib.rs]: @DDtKey I've updated the feature to use snake case everywhere.

Comment (DDtKey) [reply src/lib.rs]: Thank you!

**Review by DDtKey — APPROVED** [review]

**Pull request merged**